scripts/build/config.js

```diff
@@ -109,7 +109,9 @@ const coreBundles = [
     input: "index.js",
     type: "core",
     target: "node",
-    external: [path.resolve("src/common/third-party.js")],
+    internalReplace: {
+      [path.resolve("src/common/third-party.js")]: "./third-party"
+    },
```

Comment:
This isn't necessary, the `external` works

Reply:
No, it's not, you can try, I don't know what went wrong, a error throws, and I wasted lots of time trying to figure why

Reply:
Indeed. It's because of the `rollup-plugin-node-resolve` update